docs/scripts/generate-sitemap-robotstxt.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * Generate SiteMap for SEO Purpose
+ * We choose to write the script instead of using next-sitemap library because
+ * next-sitemap library doesn't handle Next.js dynamic path well and it's easier
+ * to maintain the small script on our own.
+ * src: https://leerob.io/blog/nextjs-sitemap-robots
+ */
+
+import dotenv from 'dotenv-safe';
+import { globby } from 'globby';
+import prettier from 'prettier';
+import { writeFileSync } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { getContentPaths } from '../src/utils/getContentPaths.js';
+import { getPageFromSlug } from '../src/utils/getPageFromSlug.js';
+import { getPagesManifest } from '../src/utils/getPagesManifest.js';
```

Comment:
Do you have to add the js extension?
```suggestion
import { getPagesManifest } from '../src/utils/getPagesManifest';
```

Reply:
Yes, we have to add the extension in this case to specify it's `js` even though it's actually `ts` file